docs/public/_redirects

```diff
@@ -9,3 +9,4 @@
 /ja/* /:splat 200
 
 # Redirections
+/api/data-grid/x-grid /api/data-grid/data-grid-pro 301
```

Comment:
This creates a redirection chain

<img width="823" alt="Capture d’écran 2021-08-19 à 22 48 03" src="https://user-images.githubusercontent.com/3165635/130141867-1e864da3-e1bd-4d6a-aa1e-c298881df6e8.png">

```suggestion
/api/data-grid/x-grid/ /api/data-grid/data-grid-pro/ 301

```